control/controlclient/auto.go

```diff
@@ -567,10 +580,10 @@ func (c *Auto) mapRoutine() {
 
 			if err != nil {
```

Comment:
Err is always non-nil at this point.

Reply:
And it was before too-- glad we can actually see that easily now! 😂